### PR TITLE
adds fieri documentation to private supermarket docs

### PR DESCRIPTION
**Pull request by nellshamrell**
Adds documentation around the Fieri feature of Supermarket.

### CODE REVIEW

File: chef_master/source/supermarket.rst [hunk 1]
```diff
@@ -97,3 +97,8 @@ For more information about the |supermarket rb| file, see :doc:`supermarket.rb <
 .. include:: ../../includes_api_cookbooks_site/includes_api_cookbooks_site.rst
 
 For more information about the |api cookbooks site|, see :doc:`Cookbooks Site API </api_cookbooks_site>`.
+
+fieri
+=====================================================
+.. include:: ../../includes_plugin_knife/includes_plugin_knife_supermarket.rst
```
Comment (robbkidd) [inline chef_master/source/supermarket.rst]: Should this be a reference to the `includes_supermarket_private_fieri.rst` file created in this PR?

File: includes_supermarket/includes_supermarket_private_fieri.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+   ['supermarket_omnibus']['config']['features'] = "tools,github,announcement,fieri"
+
+Then add the following Fieri attributes
+
+.. code-block:: ruby
+
+  ['supermarket_omnibus']['config']['fieri_key'] = "#{random string you generate}"
+  ['supermarket_omnibus']['config']['fieri_supermarket_endpoint'] = "#{your_supermarket_url}"
+  ['supermarket_omnibus']['config']['fieri_results_endpoint'] = "#{your_supermarket_url}/api/v1/cookbook-versions/evaluation
```
Comment (robbkidd) [inline includes_supermarket/includes_supermarket_private_fieri.rst]: Is `fieri_results_endpoint` included in the doc for compatibility with the 2.7.x versions, prior to 2.8.0 where we changed Fieri to use `fieri_supermarket_endpoint`?

File: chef_master/source/supermarket.rst (hunk 1 above)
Comment (nellshamrell) [inline chef_master/source/supermarket.rst]: Yes!  TY!

File: includes_supermarket/includes_supermarket_private_fieri.rst (hunk 2 above)
Comment (nellshamrell) [inline includes_supermarket/includes_supermarket_private_fieri.rst]: Good point, ty!

**Pull request merged**